Android build lint error - "Value must be ? 0 (was -2)"

## Environment
<!-- Run `react-native info` in your terminal and paste its contents here. -->
```
System:
    OS: Windows 10
    CPU: (4) x64 Intel(R) Core(TM) i5-4690K CPU @ 3.50GHz
    Memory: 1.63 GB / 7.89 GB
  Binaries:
    Node: 11.14.0 - C:\Program Files\nodejs\node.EXE
    npm: 6.9.0 - C:\Program Files\nodejs\npm.CMD
```
## Description
When I try to build android app with command `gradlew build`, I get the following lint error.
```js
Execution failed for task ':@react-native-community_slider:lint'.
> Lint found errors in the project; aborting build.

\node_modules\@react-native-community\slider\android\src\main\java\com\reactnativecommunity\slider\ReactSliderManager.java:68: Error: Value must be ? 0 (was -2) [Range]
              ViewGroup.LayoutParams.WRAP_CONTENT,
              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

```
https://github.com/react-native-community/react-native-slider/blob/master/android/src/main/java/com/reactnativecommunity/slider/ReactSliderManager.java#L68
<!--
  Describe your issue in detail. Include screenshots if needed. If this is a regression, let us know.
-->


## Reproducible Demo
Run android app build with `gradlew build` command.
<!--
  Let us know how to reproduce the issue. Include a code sample or share a project that reproduces the issue.
  Please follow the guidelines for providing a minimal example: https://stackoverflow.com/help/mcve.
-->

--- a/src/android/src/main/java/com/reactnativecommunity/slider/ReactSliderManager.java
+++ b/src/android/src/main/java/com/reactnativecommunity/slider/ReactSliderManager.java
@@ -66,9 +66,7 @@ public class ReactSliderManager extends SimpleViewManager<ReactSlider> {
         YogaMeasureMode heightMode) {
       if (!mMeasured) {
         SeekBar reactSlider = new ReactSlider(getThemedContext(), null, STYLE);
-        final int spec = View.MeasureSpec.makeMeasureSpec(
-            ViewGroup.LayoutParams.WRAP_CONTENT,
-            View.MeasureSpec.UNSPECIFIED);
+        final int spec = View.MeasureSpec.makeMeasureSpec(0, View.MeasureSpec.UNSPECIFIED);
         reactSlider.measure(spec, spec);
         mWidth = reactSlider.getMeasuredWidth();
         mHeight = reactSlider.getMeasuredHeight();

